qa/L0_client_memory_growth/client_memory_mail.py

```diff
@@ -37,10 +37,12 @@
     today = date.today().strftime("%Y-%m-%d")
     subject = "Triton Client Memory Growth " + sys.argv[1] + " Summary: " + today
     memory_graphs = glob.glob("client_memory_growth*.log")
-    html_content = "<html><head></head><body><pre style=\"font-size:11pt;font-family:Consolas;\">"
+    write_up = "<p>This test is run for both HTTP and GRPC protocols using C++ and Python test scripts. The thresholds of memory growth are set to 10MB and 1MB for C++ and Python tests individually as the max-allowed difference between mean and maximum memory usage.</p>"
```

Comment:
The threshold is not 10MB, the max difference between mean and max memory usage is 10MB